app/static/js/events/wizard/basic-details/initializer.js

```diff
@@ -32,6 +32,7 @@ function basicDetailsInit($eventDiv) {
     $eventDiv.find(".licence-help").click(function () {
         $("#licence-list").slideToggle();
     });
+    $eventDiv.find("#ticket").val(0.01);
```

Comment:
Why are you doing this ? Please don't manually set values like this. Set the value in the appropriate object and let Vue handle the rest for you